x-pack/plugins/ml/public/application/data_frame_analytics/pages/components/analytics_selector/analytics_id_selector.tsx

```diff
@@ -0,0 +1,280 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState, useEffect, useCallback } from 'react';
+
+import {
+  EuiInMemoryTable,
+  EuiButton,
+  EuiButtonEmpty,
+  EuiFlyout,
+  EuiFlyoutBody,
+  EuiFlyoutHeader,
+  EuiFlyoutFooter,
+  EuiTitle,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiTabbedContent,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { ml } from '../../../../services/ml_api_service';
```

Comment:
Please avoid using this static import. You can retrieve is from the context instead

Reply:
Thanks for taking a look! All type updates here 528e5a345693ad68f47e5d57e03ab858d8ee9295